LPPC/Plugins/topsky/GRpluginSettings.txt

```diff
@@ -17,16 +19,64 @@ GroundLabel=1
 GroundLabel_Font=EuroScope
 GroundLabel_FontStyle=1000,0,0,0
 GroundLabel_FontSize=11
-GroundLabel_ATYP=1
-GroundLabel_RMK=0
-GroundLabel_SID=1
-GroundLabel_WTC=1
-GroundLabel_STAND=1
+GroundLabel_Transparency_Bg=255
 
-AppLabel_Font=EuroScope
 
-AppLabel_AFL_VS=1
-AppLabel_GS=1
+AppLabel_Font=EuroScope
+AppLabel_Transparency_Bg=255
+
+Label_AFL=1,1,0
+Label_AFL+VS=0,0,0
+Label_AHDG=0,1,0
+Label_ARWY=0,0,0
+Label_ASSR=0,0,0
+Label_ATYP=0,1,0
+Label_CFL=1,0,0
+Label_DEP=0,0,0
+Label_DRWY=0,0,0
+Label_GS=1,1,0
+Label_REF=0,0,0
+Label_REG=0,0,0
+Label_RMK=0,1,0
+Label_SID=0,1,0
+Label_STAND=0,1,0
+Label_STAR=0,0,0
+Label_TSSR=1,1,0
+Label_TYP=0,0,0
+Label_VS=1,1,0
+Label_WTC=0,1,0
+
+
+Label=GND:DEP:0:ALRT,0,0:ASSR_E,0,1:COMM,0,0
+Label=GND:DEP:1:CALLSIGN,0,0:SID,8,0
+Label=GND:DEP:2:WTC,0,0:ATYP,2,0
+Label=GND:DEP:3:RMK,0,0
+
+Label=GND:ARR:0:ALRT,0,0:ASSR_E,0,1:COMM,0,0
+Label=GND:ARR:1:CALLSIGN,0,0:STAND,8,0
+Label=GND:ARR:2:WTC,0,0:ATYP,2,0
+Label=GND:ARR:3:RMK,0,0
+
+Label=GND:OTH:0:
+Label=GND:OTH:1:
+Label=GND:OTH:2:
+Label=GND:OTH:3:
+
+Label=GND:UNC:0:ALRT,0,0:ASSR_E,0,1:COMM,0,0
+Label=GND:UNC:1:CALLSIGN,0,0
```

Comment:
Check if this behaves properly and shows tssr on mode c and callsign on mode s, it should but idk

Reply:
well this sucks ass